packages/protocol/contracts/governance/VestingInstance.sol

```diff
@@ -0,0 +1,379 @@
+    getLockedGold().relock(index, value);
+  }
+
+  /**
+   * @notice A wrapper func for the withdraw locked gold method function
+   * @param index the index of the pending locked gold withdrawal
+   * @dev To be called only by the beneficiary of the vesting. The amount shall be withdrawn back to the vesting instance
+   */
+  function withdrawLockedGold(uint256 index) external nonReentrant onlyBeneficiary {
+    getLockedGold().withdraw(index);
+  }
+
+  /**
+   * @notice A wrapper func for the authorize vote signer account method
+   * @param signer The address of the signing key to authorize.
+   * @param v The recovery id of the incoming ECDSA signature.
+   * @param r Output value r of the ECDSA signature.
+   * @param s Output value s of the ECDSA signature.
+   * @dev To be called only by the beneficiary of the vesting. The v,r and s signature should be a signed message by the beneficiary being the authorized address
+   */
+  function authorizeVoteSigner(address signer, uint8 v, bytes32 r, bytes32 s)
+    external
+    nonReentrant
+    onlyBeneficiary
+  {
+    getAccounts().authorizeVoteSigner(signer, v, r, s);
+  }
+
+  /**
+   * @notice A wrapper func for the authorize validator signer account method
+   * @param signer The address of the signing key to authorize.
+   * @param v The recovery id of the incoming ECDSA signature.
+   * @param r Output value r of the ECDSA signature.
+   * @param s Output value s of the ECDSA signature.
+   * @dev To be called only by the beneficiary of the vesting.
+   */
+  function authorizeValidatorSigner(address signer, uint8 v, bytes32 r, bytes32 s)
+    external
+    nonReentrant
+    onlyBeneficiary
+  {
+    getAccounts().authorizeValidatorSigner(signer, v, r, s);
+  }
+
+  /**
+   * @notice A wrapper func for the authorize validator signer account method
+   * @param signer The address of the signing key to authorize.
+   * @param ecdsaPublicKey The ECDSA public key corresponding to `signer`.
+   * @param v The recovery id of the incoming ECDSA signature.
+   * @param r Output value r of the ECDSA signature.
+   * @param s Output value s of the ECDSA signature.
+   * @dev To be called only by the beneficiary of the vesting.
+   */
+  function authorizeValidatorSignerPubKey(
+    address signer,
+    bytes calldata ecdsaPublicKey,
+    uint8 v,
+    bytes32 r,
+    bytes32 s
+  ) external nonReentrant onlyBeneficiary {
+    getAccounts().authorizeValidatorSigner(signer, ecdsaPublicKey, v, r, s);
+  }
+
+  /**
+   * @notice A wrapper func for the authorize attestation signer account method
+   * @param signer The address of the signing key to authorize.
+   * @param v The recovery id of the incoming ECDSA signature.
+   * @param r Output value r of the ECDSA signature.
+   * @param s Output value s of the ECDSA signature.
+   * @dev To be called only by the beneficiary of the vesting.
+   */
+  function authorizeAttestationSigner(address signer, uint8 v, bytes32 r, bytes32 s)
```

Comment:
Ditto, as this is only needed for validators

Reply:
done